### ISSUE TITLE
Add/Replace audio track for a video

### ISSUE THREAD

**Issue by Nonnis-M** [PHP-FFMpeg/PHP-FFMpeg#330]
| Q              | A
| -------------- | ---
| Bug?           | no
| New Feature?   | maybe
| Version Used   | master
| FFmpeg Version | FFProbe N-84607-g9bcc5e8-tessus - ffmpeg version 3.2.4
| OS             | mac OSX Sierra 10.12.3

It is possibile to load an mp3 audio track and add (or replace) it to an existing video? If yes, I don't know how to do it. The audio and video formats doesn't matter for me.

Thanks.

**Comment by jens1o** [comment]
no, it's not possibe yet using prewritten filters. But you can use the SimpleFilters to complete this.

**Comment by Nonnis-M** [comment]
Do you refer to https://github.com/PHP-FFMpeg/PHP-FFMpeg/blob/master/src/FFMpeg/Filters/Audio/SimpleFilter.php ?

What I have to do to properly use it?

**Comment by jens1o** [comment]
yes, I do. If you know the ffmpeg command for this, it would be quite useful(I'm at mobile).

Basically, it is like this: https://github.com/PHP-FFMpeg/PHP-FFMpeg/blob/master/src/FFMpeg/Media/Video.php#L74

That way, you can add something that will be executed. (Use `Video#addFilter` to add it)

**Comment by Nonnis-M** [comment]
apologize for writing again but the ffmpeg cmd to mix audio and video is the follow:

ffmpeg -i input0 -i input1 -map 0:a -map 1:v -c copy output

and I have no idea about how to bind input0 and input1 (respectively the audio and video file in this sample) to real input files with my SimpleFilter. I have searched for samples in this repository but I didn't find anything useful.

**Comment by jens1o** [comment]
I'll check wether this is possible today. Thanks for providing the command, it is quite helpful.

**Comment by Romain** [comment]
Hi think this should be a new filter, like a MergeFilter.
This needs a PR.
@Nonnis-M can you provide us with the link to FFMPEG documentation of this feature?

**Comment by jens1o** [comment]
I'll do a pr once I created the "audio-muting" pr.

**Comment by EGreg** [comment]
Has this been done? How do you accomplish this?

**Comment by jens1o** [comment]
This is not yet implemented, feel free to file a Pull Request. :)
